### PR TITLE
Add image label description

### PR DESCRIPTION
**Pull request by WuJun2016**
# links
- https://github.com/harvester/harvester/issues/2057

### CODE REVIEW

File: docs/upload-image.md [hunk 1]
```diff
@@ -35,3 +35,12 @@ Currently, qcow2, raw, and ISO images are supported.
 On the **Volumes** page, click **Export Image**. Enter image name to create image.
 
 ![](./assets/export-image.png)
+
+### Image labels
+
+
+You can add a label for the image, which will help identify the operating system type more accurately.
+
+When you create an image via URL, we will try to add the system-related label automatically, if you create an image by uploading a local file, you will need to select the system type manually.
```
Comment (vickyhella) [inline docs/upload-image.md:44]:
```suggestion
If you create an image via a URL, we will add the OS type label automatically. However, if you create an image by uploading a local file, you will need to select the OS type manually.
```

**Review by vickyhella — CHANGES_REQUESTED** [review]

File: docs/upload-image.md (hunk 1 above)
Comment (WuJun2016) [reply docs/upload-image.md]: @vickyhella   Done,  PTAL. thanks.

Comment (vickyhella) [reply docs/upload-image.md]: LGTM

**Review by vickyhella — APPROVED** [review]

File: docs/upload-image.md [hunk 2]
```diff
@@ -35,3 +35,12 @@ Currently, qcow2, raw, and ISO images are supported.
 On the **Volumes** page, click **Export Image**. Enter image name to create image.
 
 ![](./assets/export-image.png)
+
+### Image labels
+
+
+You can add a label for the image, which will help identify the operating system type more accurately.
```
Comment (guangbochen) [inline docs/upload-image.md:42]:
```suggestion
You can add labels to the image, which will help identify the OS type more accurately. Additionally, you can also add any custom labels when needed.
```

File: docs/upload-image.md [hunk 3]
```diff
@@ -35,3 +35,12 @@ Currently, qcow2, raw, and ISO images are supported.
...
+
+If you create an image via a URL, we will add the OS type label automatically. However, if you create an image by uploading a local file, you will need to select the OS type manually.
```
Comment (guangbochen) [inline docs/upload-image.md:44]:
```suggestion
If you create an image from a URL, the UI will automatically recognize the OS type and image category based on the image name. However, if you created the image by uploading a local file, you will need to manually select the corresponding labels.
```

**Review by guangbochen — CHANGES_REQUESTED** [review]

File: docs/upload-image.md (hunk 3 above)
Comment (WuJun2016) [reply docs/upload-image.md]: Done, PTAL again thanks.

**Review by guangbochen — APPROVED** [review]

**Pull request merged**